### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by VickTor61**
Added tests to all models and controllers except the salesController because it didn't have a defined method

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,7 +1,39 @@
...
+
+  root to: 'sales#index'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
 end
+
+
+
+
+=begin
```
Comment (EnomaDebby) [inline config/routes.rb]: why is this here?

Comment (VickTor61) [reply config/routes.rb]: i was commenting out a piece of code

Comment (EnomaDebby) [reply config/routes.rb]: it has nothing to do with routes. Delete it. Whether it's a comment or not, things should be where they should be. Use your IDEs and notes for comments not related to the code
